datamodel/testdata.d/08_03_test_data_ovs.sql

```diff
@@ -0,0 +1,116 @@
+\set ON_ERROR_STOP true
+\connect dcsa_openapi
+
+BEGIN;
+
+INSERT INTO dcsa_im_v3_0.port_call_service_type (
+    port_call_service_type_code,
+    port_call_service_type_name,
```

Comment:
Does not respect defined maximum length varchar(30).
This applies to all `port_call_service_type_name` attributes. 
Error thrown -> `ERROR:  value too long for type character varying(30)`

Reply:
Ah, I think this was changed in a commit in master that was merged afterwards. They have all been shortened.